addons/mrp/models/mrp_workorder.py

```diff
@@ -377,26 +384,32 @@ def _refresh_wo_lines(self):
                     line_values = workorder._generate_lines_values(move, qty_to_consume - qty_already_consumed)
                     self.env['mrp.workorder.line'].create(line_values)
 
-    def _defaults_from_finished_workorder_line(self, reference_lot_lines):
-        for r_line in reference_lot_lines:
-            # see which lot we could suggest and its related qty_producing
-            if not r_line.lot_id:
-                continue
-            candidates = self.finished_workorder_line_ids.filtered(lambda line: line.lot_id == r_line.lot_id)
-            rounding = self.product_uom_id.rounding
-            if not candidates:
-                self.write({
-                    'finished_lot_id': r_line.lot_id.id,
-                    'qty_producing': r_line.qty_done,
-                })
-                return True
-            elif float_compare(candidates.qty_done, r_line.qty_done, precision_rounding=rounding) < 0:
-                self.write({
-                    'finished_lot_id': r_line.lot_id.id,
-                    'qty_producing': r_line.qty_done - candidates.qty_done,
-                })
-                return True
-        return False
+    def _defaults_from_finished_workorder_line(self, finished_wo):
+        suggested_lot = False
+        suggested_qty = finished_wo.qty_produced - self.qty_produced
+
+        if self.product_tracking != 'none':
+            for r_line in finished_wo.finished_workorder_line_ids:
+                # see which lot we could suggest and its related qty_producing
+                if not r_line.lot_id:
+                    if self.product_tracking == 'serial':
+                        suggested_qty = 1
+                    continue
```

Comment:
I think It's useless. 
At creation, the quantity is set to 1. Once the production start, all workorder lines have 1 as quantity done